devportalservice/errors.go

```diff
@@ -9,5 +9,5 @@ type NetworkError struct {
 }
 
 func (e NetworkError) Error() string {
-	return fmt.Sprintf("response %d %s", e.Status, e.Body)
+	return fmt.Sprintf("network request failed with status %d, body (%s)", e.Status, e.Body)
```

Comment:
i think it does not make sense to include the Body in the message, it is an io.Reader, it's string representation will not be useful i think.

Reply:
it is a string, in this case